sub-packages/bionemo-scdl/src/bionemo/scdl/index/row_feature_index.py

```diff
@@ -37,15 +39,16 @@ class RowFeatureIndex:
         correspondto a given row. For examples if the array is [-1, 200, 201],
         rows 0 to 199 correspond to _feature_arr[0] and 200 corresponds to
         _feature_arr[1]
-        _feature_arr: list of feature dataframes
+        _feature_arr: list of feature dictionaries
         _labels: list of labels
         _version: The version of the dataset
     """
 
     def __init__(self) -> None:
         """Instantiates the index."""
         self._cumulative_sum_index: np.array = np.array([-1])
-        self._feature_arr: List[pd.DataFrame] = []
+        self._feature_arr: List[dict] = []
```

Comment:
you now have the option of saying `list[dict[str, int]]` and that type of thing rather than `List[...]`. It's a new syntax option where you can use the base types for typing signatures you don't need to use the special types in the `import typing` library.